.github/workflows/release.yml

```diff
@@ -151,15 +142,11 @@ jobs:
               sudo \
               wget \
 
-          # Add the public key for the llvm repository to get the correct clang version
-          bash -c 'wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|apt-key add -'
-          apt-add-repository -y 'deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-15 main'
-
           sudo wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | sudo gpg --dearmor - | sudo tee /usr/share/keyrings/kitware-archive-keyring.gpg >/dev/null
           sudo echo 'deb [signed-by=/usr/share/keyrings/kitware-archive-keyring.gpg] https://apt.kitware.com/ubuntu/ jammy main' | sudo tee /etc/apt/sources.list.d/kitware.list >/dev/null
 
           apt-get update
-          apt-get -y --no-install-recommends install -t llvm-toolchain-jammy-15 clang-15 clang-tidy-15
+          apt-get -y --no-install-recommends install -t clang-15 clang-tidy-15
```

Comment:
I really think you can skip even this. The release build isn't using clang or clang-tidy as far as I know, and it will save you some CI time.